server/core/src/initServer.cpp

```diff
@@ -683,7 +683,7 @@ int initRsCommWithStartupPack(rsComm_t& rsComm, const startupPack_t& startupPack
     snprintf( tmpStr2, LONG_NAME_LEN, "%s=%d", IRODS_PROT, NATIVE_PROT );
     putenv( tmpStr2 );
 
-    rsComm.connectCnt = startupPack.connectCnt;
+    rsComm.connectCnt = startupPack.connectCnt + 1;
```

Comment:
need a comment here to explain what this is for - help the next reader.

Reply:
Done.